utils/m0reportbug

```diff
@@ -651,7 +653,7 @@ binaries() {
 
     mkdir -p $outdir
     ## Note, that we intentionally skip `m0tr.ko' as it's too big (42M).
-    for f in m0mkfs m0d m0ham m0ut m0beck m0betool ${libs[@]}; do
+    for f in m0mkfs m0d m0ham m0ut m0beck m0betool m0client m0touch ${libs[@]}; do
```

Comment:
[Suggestion]
Better to add m0cp and m0cat as well, since we are adding other motr utils here (m0client, m0touch)

Reply:
incorporated